tests/kafkatest/services/kafka/kafka.py

```diff
@@ -34,17 +34,22 @@
 class KafkaService(JmxMixin, Service):
 
     PERSISTENT_ROOT = "/mnt"
-    LOG4J_CONFIG_FILE = os.path.join(PERSISTENT_ROOT, "kafka-log4j.properties")
+    STDOUT_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
+    STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
```

Comment:
@ewencp 
This is really taking the existing functionality and assigning to variables which parallel the rough convention that is developing in other service classes